samples/powerpoint/basics/insert-image.yaml

```diff
@@ -0,0 +1,62 @@
+id: ''
+gist: ''
+source: POWERPOINT
+author: Microsoft
+name: insert-image
+description: Inserts an image to the current slide.
+script:
+  content: |
+    $('#insert').click(run);
+
+    async function run() {
+        var base64EncodedImageStr = getImageAsBase64String();
+
+        await Office.context.document.setSelectedDataAsync(base64EncodedImageStr, {
+            coercionType: Office.CoercionType.Image,
+            imageLeft: 50,
+            imageTop: 50,
+            imageWidth: 400
+        },
+            function (asyncResult) {
+                if (asyncResult.status === Office.AsyncResultStatus.Failed) {
+                    console.log(asyncResult.error.message);
+                }
+             });
+    }  
+
+    function getImageAsBase64String() {
```

Comment:
Or had an "upload" button, for local images.  I helped Reeza with something similar a couple months ago, he should have the code you need